components/display/include/display.hpp

```diff
@@ -10,6 +10,7 @@
 
 #include "base_component.hpp"
 #include "led.hpp"
+#include "logger.hpp"
```

Comment:
technicall you don't need this since the include comes from `base_component.hpp`